src/main/scala/org/broadinstitute/dsde/rawls/model/UserAuth.scala

```diff
@@ -0,0 +1,42 @@
+package org.broadinstitute.dsde.rawls.model
+
+import org.broadinstitute.dsde.rawls.model.WorkspaceAccessLevel._
+
+sealed trait UserAuthRef
+case class RawlsUserRef(userEmail: String) extends UserAuthRef
+case class RawlsGroupRef(groupName: String) extends UserAuthRef
```

Comment:
A Group Ref type is necessary because I want to be able to refer to it without having to specify its users and subgroups.  Not sure about User Ref.  That's really just for symmetry right now, but it could be useful if we add more fields to the User.